EVB-2/IS_EVB-2/src/sd_card_logger.cpp

```diff
@@ -238,9 +238,6 @@ void sd_card_logger_init(void)
     /* Init real-time clock: RTC configuration, 24-hour mode */
     rtc_set_hour_mode(RTC, 0);
 
-    /* Init real-time timer */
-    time_init();    
```

Comment:
This should happen only in the system init, but wasn't a problem because that function has a static int that blocks multi-time execution.